pkgs/development/tools/misc/stlink/default.nix

```diff
@@ -1,4 +1,13 @@
-{ lib, stdenv, fetchFromGitHub, cmake, libusb1 }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libusb1
+, gtk3
+, pkgconfig
```

Comment:
```suggestion
, pkg-config
```

Reply:
Thank you for the explanation. I wondered about that failure. I'll fix.